apps/web/src/services/advanced/components/WorkspaceGroupTabGroupUser.vue

```diff
@@ -16,10 +27,41 @@ const tableState = reactive({
         // TODO: temp data
         user_id: 'Kara_Herzog@yahoo.com',
         name: '',
-        state: 'Enabled',
-        role: 'Workspace Member Role',
+        state: 'ENABLED',
+        role: 'WORKSPACE_OWNER',
     }],
 });
+
+const selectDropdownState = reactive({
+    items: [{
+        label: 'Workspace Member',
+        name: 'managed-workspace-member',
+        role_type: 'WORKSPACE_MEMBER',
+    },
+    {
+        label: 'Workspace Member',
+        name: 'managed-workspace-member',
+        role_type: 'WORKSPACE_MEMBER',
+    },
+    {
+        label: 'Workspace Member',
+        name: 'managed-workspace-member',
+        role_type: 'WORKSPACE_MEMBER',
+    },
+    {
+        label: 'show more',
+        type: 'showMore',
+    },
```

Comment:
suggenstion: how about using `ROLE_TYPE. WORKSPACE_MEMBER`... instead of using 'WORKSPACE_MEMBER'?
`ROLE_TYPE` constant is in `....src/schema/identity/role/constant.ts`.